<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Withdrawal</class>
 <widget class="QWidget" name="Withdrawal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1665</width>
    <height>1043</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>HP Simplified</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelWithdrawal">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>1001</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>46</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Syötä nostettava summa</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClearWithdrawal">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>320</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Tyhjennä valinnat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDoWithdrawal">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>590</y>
     <width>650</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nosta rahat</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWithdrawalMessage">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLogout">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>270</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAdd20">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>150</width>
     <height>125</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>+20€</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAdd50">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>430</y>
     <width>150</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+50€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAdd500">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>430</y>
     <width>150</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+500€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonReturnMenu">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>470</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Päävalikko</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAdd100">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>430</y>
     <width>150</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+100€</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textWithdrawal">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>260</y>
     <width>320</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'HP Simplified'; font-size:50pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
